### DISCUSSION TITLE
Navigation on items within a collection

### DISCUSSION THREAD

**Discussion by igorricardo20** [directus/directus#20314] [Feature Requests]
### Summary

There could be two arrows in the Item page that allows the user to navigate among the collection items. 

### Basic Example

<img width="904" alt="image" src="https://github.com/directus/directus/assets/18128219/73b5b876-987f-4699-9382-39d31d5bd52e">

### Motivation

This could help a lot when dealing with a large collection, because we would not need to go back and further every time

**Comment by rijkvanzanten** [comment]
What would determine which one is "next" or "previous"?

**Reply by Boegie19** [reply]
The order they are currently at in the content page and for relations you take the order the relations are displayed in.

**Comment by jshoemaker-gofer** [comment]
I came here to request this same feature. I voted for this. 

The "next" record should be the next record in the collection list **as currently sorted**. This part does make this request a little more complicated, I would imagine. 

I would add that it would be nice if the current location of the page would stay put, too. This way the same fields would be visible for each record. This would allow us to easily click through and see the same field for each item instead of scrolling each time.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
